recipes/rust-bio-tools/meta.yaml

```diff
@@ -23,13 +23,13 @@ requirements:
     - zlib
     - clangdev
     - gsl
-    - openblas
+    - libopenblas
```

Comment:
If you don't depend on this specific BLAS implementation, you could also just use `libblas` here, see https://conda-forge.org/docs/maintainer/knowledge_base.html#knowledge-blas.